consensus/istanbul/core/core.go

```diff
@@ -304,7 +304,7 @@ func (c *core) catchUpRound(view *istanbul.View) {
 	c.roundChangeSet.Clear(view.Round)
 
 	c.newRoundChangeTimer()
-	logger.Trace("Catch up round", "new_round", view.Round, "new_seq", view.Sequence, "new_proposer", c.valSet)
```

Comment:
This is not about log level, but it seems that the `new_proposer` and `c.valSet` are not matched.

Reply:
@jeongkyun-oh Let's handle the issue in another PR or other channel.

Reply:
👌